src/lib/button-toggle/button-toggle.ts

```diff
@@ -21,7 +21,8 @@ import {
     FormsModule,
 } from '@angular/forms';
 import {Observable} from 'rxjs/Observable';
-import {BooleanFieldValue, MdUniqueSelectionDispatcher} from '@angular2-material/core';
+import {MdUniqueSelectionDispatcher} from '@angular2-material/core';
+import {coerceBooleanProperty} from '../core/coersion/boolean-property';
```

Comment:
Either wait for my PR to be merged in (where you can use `../core`) or fix those imports to pass the linting.